src/Detector/Hugo/HugoDetector.cs

```diff
@@ -155,23 +167,25 @@ private bool IsHugoApp(ISourceRepo sourceRepo, out string appDirectory)
         private bool IsHugoTomlFile(ISourceRepo sourceRepo, params string[] subPaths)
         {
             TomlTable tomlTable = null;
-
-            var relativeFilePath = Path.Combine(subPaths);
-
-            try
-            {
-                tomlTable = ParserHelper.ParseTomlFile(sourceRepo, relativeFilePath);
-            }
-            catch (FailedToParseFileException ex)
+            foreach (string subPath in subPaths)
```

Comment:
I'm not sure I understand the need for the addition of this `foreach` block -- would you mind elaborating?